Dockerfile

```diff
@@ -0,0 +1,19 @@
+#FROM nvidia/cuda:11.8.0-base-ubuntu22.04
+FROM python:3.9
```

Comment:
You can use [`ARG`](https://docs.docker.com/engine/reference/builder/#arg) to pass the base image